mindmeld/components/entity_resolver.py

```diff
@@ -603,18 +372,17 @@ def fit(self, clean=False):
         self.ready = True
         return
 
-    @abstractmethod
-    def _predict(self, nbest_entities, top_n):
-        raise NotImplementedError
-
-    def predict(self, entity, top_n: int = 20):
+    def predict(self, entity, top_n=20):
```

Comment:
Not sure why this line was changed `def predict(self, entity, top_n: int = 20):` seems ok